server/grpc_test.go

```diff
@@ -92,7 +92,7 @@ func TestMain(m *testing.M) {
 	listener = bufconn.Listen(bufSize)
 
 	validateAC := true
-	mangleACKeys := false
+	mangleACKeys := true
```

Comment:
This now means that we aren't testing the regular non-mangled AC key case. I will refector the grpc tests in #594 to avoid this.